content/guides/nextjs/github/configure-custom-app.md

```diff
@@ -32,15 +41,14 @@ export default class Site extends App {
         /**
          * 2. Register the GithubClient
          */
-        github: new GithubClient({
-          proxy: '/api/proxy-github',
-          authCallbackRoute: '/api/create-github-access-token',
-          clientId: process.env.GITHUB_CLIENT_ID,
-          baseRepoFullName: process.env.REPO_FULL_NAME, // e.g: tinacms/tinacms.org,
-        }),
+        github,
       },
       /**
-       * 3. Use the Sidebar and Toolbar
+       * 3. Register the Media Store
+       */
+      media: new GithubMediaStore(github),
```

Comment:
```suggestion
      media: new NextGithubMediaStore(github),
```

Why not use the NextJS Media Store here?

Reply:
I'll add this as a tip!